hubstorage/batchuploader.py

```diff
@@ -250,17 +252,21 @@ def __str__(self):
 
 
 def _encode_identity(iter):
-    data = StringIO()
+    data = BytesIO()
     for item in iter:
```

Comment:
sorry for nitpicking, but can we replace `iter` with something like `iterable` here - I don't like when builtins are shadowed